README.md

```diff
@@ -1 +1,67 @@
-# To-Do-App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do App
+
+Web app for creating to-do lists. Each to-do has a task, description, due date and priority. The to-do list belong to a project. The user
+can create and delete different projects to store different lists. After a task is done the user can mark it, and take it off the list. App
+created using Javascript Classes, Modules and Webpack. 
+
+![screenshot](/screenshot.png)
+
+## Built With
+
+- Javascript
+- Webpack
+- Bootstrap
+
+## Live Demo
+
+[Live Demo Link](https://thneves.github.io/To-Do-App/)
```

Comment:
- [ ] This live link should be active before the project gets approved. Do you mind fixing this? Thank you